examples/csmith/Makefile

```diff
@@ -0,0 +1,32 @@
+
+SOURCES=$(wildcard *.c)
+SOURCENAMES=$(basename $(SOURCES))
+RELFS=$(addsuffix .relf,$(SOURCENAMES))
+ADTS=$(addsuffix .adt,$(SOURCENAMES))
+OUTS=$(addsuffix .out,$(SOURCENAMES))
+CHECKSUMS=$(addsuffix .checksum,$(SOURCENAMES))
+
+CSMITH_HEADER = "/nix/store/6rawnpny818v7wki6zlr3i0f3phmysyb-csmith-2.3.0/include/csmith-2.3.0/"
```

Comment:
is hardcoding this a good idea/going to be useful to anyone else?

Reply:
No, It should be at least the normal install location /usr/include/csmith